README.md

```diff
@@ -28,7 +28,7 @@ A bundle will be produced from the native packager settings of this project. A b
 packager distribution and includes some component configuration. To load the bundle:
 
 ```bash
-loadBundle <HIT THE TAB KEY AND THEN RETURN>
+conductr:loadBundle <HIT THE TAB KEY AND THEN RETURN>
```

Comment:
If we're going to change anything in terms of the structure of the commands then we should align more closely with the Python CLI - there's a ticket for this.

Thus the above command should be `conduct load`.